internal/uvm/scsi.go

```diff
@@ -62,21 +63,35 @@ func (uvm *UtilityVM) findSCSIAttachment(findThisHostPath string) (int, int32, s
 	return -1, -1, "", ErrNotAttached
 }
 
-// AddSCSI adds a SCSI disk to a utility VM at the next available location.
-// This function should be called for a RW/scratch layer. For read-only layers
-// on LCOW as an alternate to PMEM for large layers, use AddSCSILayer instead.
+// AddSCSI adds a SCSI disk to a utility VM at the next available location. This
+// function should be called for a RW/scratch layer or a passthrough vhd/vhdx.
+// For read-only layers on LCOW as an alternate to PMEM for large layers, use
+// AddSCSILayer instead.
 //
-// hostPath is required
-// uvmPath is optional.
+// `hostPath` is required and must point to a vhd/vhdx path.
+// `uvmPath` is optional.
 func (uvm *UtilityVM) AddSCSI(hostPath string, uvmPath string) (int, int32, error) {
-	return uvm.addSCSIActual(hostPath, uvmPath, false)
+	return uvm.addSCSIActual(hostPath, uvmPath, false, false)
+}
+
+// AddSCSIPhysicalDisk attaches a physical disk from the host directly to the
+// Utility VM at the next available location.
+//
+// `hostPath` is required and `must` start with `\\.\PHYSICALDRIVE`. `uvmPath`
+// is optional if a guest mount is not requested.
+func (uvm *UtilityVM) AddSCSIPhysicalDisk(hostPath, uvmPath string) (int, int32, error) {
+	prefix := `\\.\PHYSICALDRIVE`
+	if !strings.HasPrefix(hostPath, prefix) {
```

Comment:
Is this check strictly necessary? I would hope the virt stack would error out for us.

If it is necessary, we can probably find a better check that doesn't depend on the path.

Reply:
No I will remove. If the platform doesn't give a good error message here I will put back.